q2_fmt/_peds.py

```diff
@@ -466,11 +479,458 @@ def _create_masking(time_metadata, donor_df, recip_df, reference_column):
     donor_df = donor_df.to_numpy()
     donor_mask = donor_df[donor_index_masking]
     donor_mask = donor_mask.astype(int)
-    recipdf = recip_df.to_numpy()
-    recipdf = recipdf.astype(int)
     return donor_mask
 
 
 def _mask_recipient(donor_mask, recip_df):
-    maskedrecip = donor_mask & recip_df
-    return maskedrecip
+    recip_mask = donor_mask & recip_df
+    return recip_mask
+
+
+def peds_simulation(table: pd.DataFrame, metadata: qiime2.Metadata,
+                    time_column: str, reference_column: str,
+                    subject_column: str,
+                    filter_missing_references: bool = False,
+                    drop_incomplete_subjects: bool = False,
+                    drop_incomplete_timepoint: list = None,
+                    iterations: int = 999) -> (pd.DataFrame, pd.DataFrame):
```

Comment:
I would call this `n` or `num_iterations`.

Reply:
I went with num_iterations.